hardhat/test/00-ape-protocol/00-ape-vault-factory.ts

```diff
@@ -1,10 +1,15 @@
-import { expect } from 'chai';
+import chai from 'chai';
```

Comment:
you overwrote some of my changes here--please look at https://github.com/coordinape/coordinape/pull/248/commits/2539a5bb2cff9b219abb73090e9e2962ef613266

Reply:
If I don't use middleware, the linter will give linting error and also won't give appropriate suggestions
<img width="691" alt="Screenshot 2021-11-06 at 1 01 48 PM" src="https://user-images.githubusercontent.com/51387861/140601828-5d09064d-af42-420b-95fc-4ea117d3c37e.png">

Reply:
NVM, I see hardhat-waffle will take care of this. I was just using ethereum-waffle before.

Reply:
When we use hardhat-waffle, hardhat inject this implicitly and linter and IDE can't recognise it that's why it's showing error